include/deal.II/hp/fe_collection.h

```diff
@@ -276,30 +274,70 @@ namespace hp
+     * indexed by the numbers provided through @p fes . In other words, we
+     * first form the set of elements in this collection that dominate
+     * all of the ones that are indexed by the argument @p fes, and then
+     * within that set of dominating elements, we find the <i>least</i>
+     * dominant one.
+     *
+     * For example, if an FECollection consists of
+     * `{FE_Q(1),FE_Q(2),FE_Q(3),FE_Q(4)}` elements
+     * and the argument @p fes equals `{2,3}`, then the set of dominating
+     * elements consists of `{0,1,2}`, of which `2` (i.e., the `FE_Q(3)`) is the
+     * least dominant one, and then that's what the function returns.
+     *
+     * On the other hand, if the FECollection consists of
+     * `{FE_Q(1)xFE_Q(1),FE_Q(2)xFE_Q(2),FE_Q(2)xFE_Q(3),FE_Q(3)xFE_Q(2)}`
+     * elements and the argument is again @p fes equal to `{2,3}`, then the set of dominating
+     * elements consists of `{0,1}` because now neither of the last two
+     * elements dominates the other, of which `1` (i.e., the `FE_Q(2)xFE_Q(2)`)
+     * is the least dominant one -- so that's what the function returns in
+     * this case.
+     *
+     * For the purpose of this function by domination we consider either
+     * FiniteElementDomination::Domination::this_element_dominates or
+     * FiniteElementDomination::Domination::either_element_can_dominate;
+     * therefore the element can dominate itself. Thus, if an FECollection
+     * contains `{FE_Q(1),FE_Q(2),FE_Q(3),FE_Q(4)}` and @p fes only has
+     * a single element `{3}`, then the function returns 3.
      *
-     * @deprecated This function has been renamed. Use
-     * hp::FECollection::find_least_face_dominating_fe_in_collection() instead.
+     * If the function is not able to find a finite element that satisfies
+     * the description above, the function returns
+     * numbers::invalid_unsigned_int. An example would go like this:
+     * If the FECollection consists of `{FE_Nothing x FE_Nothing, FE_Q(1)xFE_Q(2), FE_Q(2)xFE_Q(1)}` with @p fes as `{1}`,
+     * the function will not find a most dominating element as the default
+     * behavior of FE_Nothing is to return
+     * FiniteElementDomination::no_requirements when comparing for face
+     * domination with any other element. In other words, the set of dominating
+     * elements is empty, and we can not find a least dominant one among it. The
+     * return value is therefore numbers::invalid_unsigned_int.
+     *
+     * The @p codim parameter describes the codimension of of the investigated
+     * subspace and specifies that it is subject to this comparison.
      */
-    DEAL_II_DEPRECATED unsigned int
-    find_least_face_dominating_fe(const std::set<unsigned int> &fes) const;
+    unsigned int
+    find_least_dominating_fe_in_collection(const std::set<unsigned int> &fes,
+                                           const unsigned int codim = 0) const;
```

Comment:
You're removing a deprecated function here but it hasn't been deprecated for a full release. Can you put it back with the obvious implementation calling the new function?

Reply:
The function `find_least_face_dominating_fe` is still available, but deprecated now: https://github.com/dealii/dealii/blob/0cc6940dfcc073eacbb7ac2dbfd8510206ac70c4/include/deal.II/hp/fe_collection.h#L291

We supplied a new name for this function last week, which is `find_least_face_dominating_fe_in_collection`, which now has an additional parameter `codim`. I didn't bother and just added that parameter because this function was added just recently. Or do you consider this a problem?

Reply:
No, that's fine. I just didn't see the old function.